test/features/build_server/compiler_interface_test.dart

```diff
@@ -264,7 +264,7 @@ Future<void> main() async {
       final receivePort = ReceivePort();
       final sendPort = testReceive.sendPort;
       final testSend = receivePort.sendPort;
-      final buildCompleter = Completer<void>();
+      final buildCompleter = VHook<void>.empty();
```

Comment:
The prefered style of `.empty()` `VHook`'s is not specifying a type thus far, so please remove the `void`.

```suggestion
      final buildCompleter = VHook.empty();
```